srcpkgs/tidy5/template

```diff
@@ -6,13 +6,13 @@ wrksrc="tidy-html5-${version}"
 build_style=cmake
 configure_args="-DTIDY_CONSOLE_SHARED=ON"
 cmake_builddir=build/cmake
-hostmakedepends="libxslt"
+hostmakedepends="libxslt grep"
```

Comment:
why is grep needed?

Reply:
*] Updating repository `https://ftp.swin.edu.au/voidlinux/current/multilib/nonfree/x86_64-repodata' ...
=> xbps-src: updating software in / masterdir...
/void-packages/common/xbps-src/shutils/chroot.sh: line 51: grep: command not found
=> xbps-src: cleaning up / masterdir...

build breaks further down as it doesnt have grep.   I have bootstrapped masterdir. Im now wondering if the mirror is missing files?

Reply:
https://github.com/void-linux/void-packages/pull/39575#discussion_r985083246

Reply:
fixed my masterdir.
removed grep.